modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/deploy/DeployRequestBase.java

```diff
@@ -95,4 +95,7 @@ public class DeployRequestBase implements Serializable {
 
     @Schema(description = "The availability zones to deploy the service instance.")
     private Map<String, String> availabilityZones;
+
+    @Schema(description = "The mark of whether to accept Eula’s services.")
```

Comment:
description can be - The flag to determine if the user has accepted the EULA terms.